ci_framework/roles/dlrn_promote/tasks/check_reported_jobs.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Check reports from DLRN
+  ansible.builtin.shell:
+    cmd: >-
+      set -o pipefail;
+      export SSL_CA_BUNDLE={{ cifmw_dlrn_promote_ssl_ca_bundle }};
+      dlrnapi --url {{ cifmw_repo_setup_dlrn_api_url }}
+      {% if cifmw_dlrn_promote_dlrnapi_host_principal is defined and cifmw_dlrn_promote_kerberos_auth|bool -%}
+      --server-principal {{ cifmw_dlrn_promote_dlrnapi_host_principal }} --auth-method kerberosAuth --force-auth
+      {% endif -%}
+      repo-status
+      {% if (cifmw_repo_setup_full_hash is defined) and (cifmw_repo_setup_full_hash | length > 0) -%}
+      --agg-hash {{ cifmw_repo_setup_full_hash }}
+      {% endif -%}
+      {% if (cifmw_repo_setup_extended_hash is defined) and (cifmw_repo_setup_extended_hash | length > 0) -%}
+      --extended_hash {{ cifmw_repo_setup_extended_hash }}
+      {% endif -%}
+      {% if (cifmw_repo_setup_commit_hash is defined) and (cifmw_repo_setup_commit_hash | length > 0) -%}
+      --commit_hash {{ cifmw_repo_setup_commit_hash }}
+      {% endif -%}
+      {% if (cifmw_repo_setup_distro_hash is defined) and (cifmw_repo_setup_distro_hash | length > 0) -%}
+      --distro_hash {{ cifmw_repo_setup_distro_hash }}
```

Comment:
would be nice if we could find a way to avoid the repetition here and in ci_framework/roles/dlrn_promote/tasks/promote_hash.yml
 and ci_framework/roles/dlrn_promote/tasks/check_for_previous_promotions.yml

Reply:
Good idea, I think we can do a follow up here.